components/brave_wallet_ui/components/desktop/portfolio-asset-item/style.ts

```diff
@@ -134,3 +128,27 @@ export const AssetMenuButtonIcon = styled(Icon).attrs({
   --leo-icon-size: 24px;
   color: ${leo.color.icon.default};
 `
+
+export const Wrapper = styled(Column)<{
+  showBorder?: boolean
+}>`
+  border-radius: ${(p) => (p.showBorder ? 16 : 0)}px;
+  border: ${(p) =>
+    p.showBorder ? `1px solid ${leo.color.divider.subtle}` : 'none'};
+`
+
+export const InfoBar = styled(Row)`
+  background-color: ${leo.color.systemfeedback.infoBackground};
+  border-radius: 12px;
+  padding: 8px 16px;
+`
+
+export const InfoText = styled(Text)`
+  line-height: 22px;
+  color: ${leo.color.systemfeedback.infoText};
+`
+
+export const LoadingRing = styled(ProgressRing)`
+  --leo-progressring-size: 20px;
+  margin-right: 16px;
```

Comment:
can we set a `gap` on the row that contains this instead?

Reply:
Done!